### PR TITLE
update: update IDE support message

### PR DESCRIPTION
**Pull request by koshachy**

### CODE REVIEW

File: docs/topics/faq.md [hunk 1]
```diff
@@ -72,14 +72,11 @@ For more information, check out the [Kotlin/Native documentation](native-overvie
...
-
-> Kotlin Plugin for Eclipse has limited support.  
-> Use [other IDEs for Kotlin development](kotlin-ide.md) to get full support of new Kotlin features.
->
-{type="warning"}
+Kotlin has full out-of-the-box support in [IntelliJ IDEA](https://www.jetbrains.com/idea/download/) and
+[Android Studio](https://developer.android.com/kotlin/get-started) by official Kotlin plugin developed by JetBrains.
+Other IDEs and source editors, such as Eclipse, Visual Studio Code, Atom, have Kotlin community-supported plugins in marketplaces.
+
+In addition, a [command line compiler](command-line.md) is available and provides straightforward support for compiling and running applications.
```
Comment (elizarov) [inline docs/topics/faq.md:79]: It is worth giving a link to the Kotlin playground here, _before_ mentioning command line, as many IDE-less people would actually prefer an online playgaround vs having to go through command-line version.

File: docs/topics/faq.md [hunk 2]
```diff
@@ -72,14 +72,11 @@ For more information, check out the [Kotlin/Native documentation](native-overvie
...
-[Android Studio](https://developer.android.com/kotlin/get-started), and [Eclipse](https://www.eclipse.org/downloads/). In addition, a [command line compiler](command-line.md) 
-is available and provides straightforward support for compiling and running applications.
-
-> Kotlin Plugin for Eclipse has limited support.  
-> Use [other IDEs for Kotlin development](kotlin-ide.md) to get full support of new Kotlin features.
->
-{type="warning"}
+Kotlin has full out-of-the-box support in [IntelliJ IDEA](https://www.jetbrains.com/idea/download/) and
+[Android Studio](https://developer.android.com/kotlin/get-started) by official Kotlin plugin developed by JetBrains.
+Other IDEs and source editors, such as Eclipse, Visual Studio Code, Atom, have Kotlin community-supported plugins in marketplaces.
```
Comment (elizarov) [inline docs/topics/faq.md:77]: I'd drop "in marketplaces" here. Where are those community-supported plugins is not relevant and might vary over time.

File: docs/topics/kotlin-ide.md [hunk 3]
```diff
@@ -1,11 +1,9 @@
...
-[//]: # (description: JetBrains provides Kotlin plugin support for IntelliJ IDEA, Android Studio, and Eclipse)
+[//]: # (description: JetBrains provides Kotlin plugin support for IntelliJ IDEA and Android Studio.)
 
-JetBrains provides the official Kotlin plugin for the following Integrated Development Environments (IDEs):
+JetBrains provides the official Kotlin plugin for two Integrated Development Environments (IDEs): [IntelliJ IDEA](#intellij-idea) and [Android Studio](#android-studio).
 
-* [IntelliJ IDEA](#intellij-idea)
-* [Android Studio](#android-studio)
-* [Eclipse](#eclipse)
+Other IDEs and source editors, such as Eclipse, Visual Studio Code, Atom, have Kotlin community-supported plugins in marketplaces.
```
Comment (elizarov) [inline docs/topics/kotlin-ide.md:6]: Same here. Let's drop "in marketplaces".

**Review by elizarov — CHANGES_REQUESTED** [review]

File: docs/topics/faq.md (hunk 1 above)
Comment (koshachy) [reply docs/topics/faq.md:82]: True, I've updated the section

**Review by elizarov — APPROVED** [review]

File: docs/topics/faq.md [hunk 4]
```diff
@@ -72,14 +72,14 @@ For more information, check out the [Kotlin/Native documentation](native-overvie
...
-Kotlin is supported by all major Java IDEs including [IntelliJ IDEA](https://www.jetbrains.com/idea/download/),
-[Android Studio](https://developer.android.com/kotlin/get-started), and [Eclipse](https://www.eclipse.org/downloads/). In addition, a [command line compiler](command-line.md) 
-is available and provides straightforward support for compiling and running applications.
-
-> Kotlin Plugin for Eclipse has limited support.  
-> Use [other IDEs for Kotlin development](kotlin-ide.md) to get full support of new Kotlin features.
->
-{type="warning"}
+Kotlin has full out-of-the-box support in [IntelliJ IDEA](https://www.jetbrains.com/idea/download/) and
+[Android Studio](https://developer.android.com/kotlin/get-started) by official Kotlin plugin developed by JetBrains.
```
Comment (danil-pavlov) [inline docs/topics/faq.md:76]:
```suggestion
[Android Studio](https://developer.android.com/kotlin/get-started) with an official Kotlin plugin developed by JetBrains.
```

File: docs/topics/faq.md [hunk 5]
```diff
@@ -72,14 +72,14 @@ For more information, check out the [Kotlin/Native documentation](native-overvie
...
+Other IDEs and source editors, such as Eclipse, Visual Studio Code, Atom, have Kotlin community-supported plugins.
```
Comment (danil-pavlov) [inline docs/topics/faq.md:77]:
```suggestion
Other IDEs and source editors, such as Eclipse, Visual Studio Code, and Atom, have Kotlin community-supported plugins.
```

File: docs/topics/faq.md [hunk 6]
```diff
@@ -72,14 +72,14 @@ For more information, check out the [Kotlin/Native documentation](native-overvie
...
+
+You can also try [Kotlin Playground](https://play.kotlinlang.org) for writing, running, and sharing 
+Kotlin code in your browser.
+
+In addition, a [command line compiler](command-line.md) is available and provides straightforward support for compiling and running applications.
```
Comment (danil-pavlov) [inline docs/topics/faq.md:82]:
```suggestion
In addition, a [command line compiler](command-line.md) is available, which provides straightforward support for compiling and running applications.
```

File: docs/topics/kotlin-ide.md [hunk 7]
```diff
@@ -1,11 +1,9 @@
...
-[//]: # (description: JetBrains provides Kotlin plugin support for IntelliJ IDEA, Android Studio, and Eclipse)
+[//]: # (description: JetBrains provides Kotlin plugin support for IntelliJ IDEA and Android Studio.)
 
-JetBrains provides the official Kotlin plugin for the following Integrated Development Environments (IDEs):
+JetBrains provides the official Kotlin plugin for two Integrated Development Environments (IDEs): [IntelliJ IDEA](#intellij-idea) and [Android Studio](#android-studio).
 
-* [IntelliJ IDEA](#intellij-idea)
-* [Android Studio](#android-studio)
-* [Eclipse](#eclipse)
+Other IDEs and source editors, such as Eclipse, Visual Studio Code, Atom, have Kotlin community-supported plugins.
```
Comment (danil-pavlov) [inline docs/topics/kotlin-ide.md:6]:
```suggestion
Other IDEs and source editors, such as Eclipse, Visual Studio Code, and Atom, have Kotlin community-supported plugins.
```

File: docs/topics/kotlin-ide.md [hunk 8]
```diff
@@ -25,34 +23,19 @@ Kotlin plugin is bundled with each Android Studio release.
...
 When the new Kotlin version is released, these IDEs will suggest updating Kotlin to the latest version automatically.
 See the latest supported language version for each IDE in [Kotlin releases](releases.md#ide-support)
 
-The Kotlin plugin for Eclipse should be installed manually.
-
 ## Other IDEs support
 
 JetBrains doesn't provide the Kotlin plugin for other IDEs.
-This means that you can use any editor to write the Kotlin code, but without IDE related features: code formatting, debugging tools, and others.
+However, some of other IDEs and source editors, such as Eclipse, Visual Studio Code, Atom, have own Kotlin plugins which developed by Kotlin community.
```
Comment (danil-pavlov) [inline docs/topics/kotlin-ide.md:35]:
```suggestion
However, some of the other IDEs and source editors, such as Eclipse, Visual Studio Code, and Atom, have their own Kotlin plugins developed by the Kotlin community.
```

File: docs/topics/kotlin-ide.md [hunk 9]
```diff
@@ -25,34 +23,19 @@ Kotlin plugin is bundled with each Android Studio release.
...
 
-To use Kotlin in other IDEs or text editors, you can download the latest Kotlin command-line compiler (`kotlin-compiler-%kotlinVersion%.zip`) from Kotlin [GitHub Releases](%kotlinLatestUrl%) and [install it manually](command-line.md#manual-install).
+You can use any text editor to write the Kotlin code, but without IDE related features: code formatting, debugging tools, and others.
```
Comment (danil-pavlov) [inline docs/topics/kotlin-ide.md:37]:
```suggestion
You can use any text editor to write the Kotlin code, but without IDE-related features: code formatting, debugging tools, and so on.
```

**Pull request merged**